bambooflow/datapipes/__init__.py

```diff
@@ -6,4 +6,7 @@
 network connections, or performing read operations on multiple files at once.
 """
 
-from bambooflow.datapipes.aiter import AsyncIterDataPipe
+from bambooflow.datapipes.aiter import (
+    AsyncIterDataPipe,
+    AsyncIterableWrapperAsyncIterDataPipe as AsyncIterableWrapper,
```

Comment:
This AsyncIterableWrapperAsyncIterDataPipe class name is a mouthful, might need to reconsider the naming scheme at some point.